packages/react-sdk/src/components/image-dev.stories.tsx

```diff
@@ -0,0 +1,200 @@
+  React.ForwardRefExoticComponent<
+    ImageProps & { style: HTMLAttributes<"img">["style"] }
+  >
+> = (args) => {
+  const style = { maxWidth: "100%", ...args.style };
+
+  return (
+    <ImagePrimitive
+      {...args}
+      optimize={true}
+      loader={imageLoader}
+      style={style}
+    />
+  );
+};
+
+export const FixedWidthImage = ImageTemplate.bind({});
+
+FixedWidthImage.parameters = {
+  docs: {
+    description: {
+      story: "Load images depending on image width and device per pixel ratio.",
+    },
+  },
+};
+
+FixedWidthImage.args = {
+  src: imageSrc,
+  width: 400,
+  height: 400,
+};
+
+export const FixedWidthImageCover = ImageTemplate.bind({});
+
+FixedWidthImageCover.parameters = {
+  docs: {
+    description: {
+      story:
+        "Preserve ratio using object-fit: cover. Load images depending on image width and device per pixel ratio.",
+    },
+  },
+};
+
+FixedWidthImageCover.args = {
+  src: imageSrc,
+  width: 400,
+  height: 400,
+  style: { objectFit: "cover" },
+};
+
+export const UnknownWidthImage = ImageTemplate.bind({});
+
+UnknownWidthImage.parameters = {
+  docs: {
+    description: {
+      story: `Load images depending on the viewport width.`,
+    },
+  },
+};
+
+UnknownWidthImage.args = {
+  src: imageSrc,
+};
+
+FixedWidthImageCover.args = {
+  src: imageSrc,
+  width: 400,
+  height: 400,
+  style: { objectFit: "cover" },
+};
+
+export const AspectRatioImage = ImageTemplate.bind({});
+
+AspectRatioImage.parameters = {
+  docs: {
+    description: {
+      story: `Fit width of the parent container, has own aspect-ratio and object-fit=cover. Load images depending on the viewport width.`,
+    },
+  },
+};
+
+AspectRatioImage.decorators = [
+  (Story) => <div style={{ maxWidth: "400px" }}>{<Story />}</div>,
+];
+
+AspectRatioImage.args = {
+  src: imageSrc,
+  style: { aspectRatio: "2/1", objectFit: "cover", width: "100%" },
+};
+
+export const FillParentImage = ImageTemplate.bind({});
+
+FillParentImage.parameters = {
+  docs: {
+    description: {
+      story: `Fill width and height of the relative parent container, object-fit=cover. Load images depending on the viewport width.`,
+    },
+  },
+};
+
+FillParentImage.decorators = [
+  (Story) => (
+    <div
+      style={{ maxWidth: "400px", aspectRatio: "2/1", position: "relative" }}
+    >
+      {<Story />}
+    </div>
+  ),
+];
+
+FillParentImage.args = {
+  src: imageSrc,
+  style: {
+    position: "absolute",
+    inset: 0,
+    width: "100%",
+    height: "100%",
+    objectFit: "cover",
+  },
+};
+
+export const HeroImage = ImageTemplate.bind({});
+
+HeroImage.parameters = {
+  docs: {
+    description: {
+      story: `"sizes" attribute explicitly equal to 100vw allowing to skip the default behavior. See DEFAULT_SIZES in the Image component. Load images depending on the viewport width.`,
+    },
+  },
+};
+
+HeroImage.args = {
+  src: imageSrc,
+  sizes: "100vw",
+  style: { aspectRatio: "3/4", objectFit: "cover", width: "100%" },
+};
```

Comment:
btw the good thing about stories is that we also have visual regression tests and this is gonna run on ci and ensure the images look the same